### PR TITLE
fix(ui): Fix failing tests and change feature flag

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

This PR makes a few changes:

1. Fixes a failing CI test by adding feature flag logic to a test
2. Changed the feature flag name from `ROX_VULN_MGMT_2_MISC_IMPROVEMENTS` to `ROX_VULN_MGMT_2_GA`
3. Added a feature flag for `ROX_VULN_MGMT_ADVANCED_FILTERS `

### CODE REVIEW

File: pkg/features/list.go [hunk 1]
```diff
@@ -110,6 +110,9 @@ var (
 	// support it but now appear to (ie: Nexus and RHEL).
 	AttemptManifestDigest = registerFeature("Enables attempts to pull manifest digests for all registry integrations", "ROX_ATTEMPT_MANIFEST_DIGEST", true)
 
-	// VulnMgmt2MiscImprovements enables support for misc improvements for VM 2.0 GA
-	VulnMgmt2MiscImprovements = registerFeature("Enables support for misc improvements for VM 2.0 GA", "ROX_VULN_MGMT_2_MISC_IMPROVEMENTS", true)
+	// VulnMgmt2GA enables support for migration changes for VM 2.0 GA
+	VulnMgmt2GA = registerFeature("Enables support for migration changes for VM 2.0 GA", "ROX_VULN_MGMT_2_GA", true)
+
+	// VulnMgmtAdvancedFilters enables support for advanced filters for VM 2.0 GA
+	VulnMgmtAdvancedFilters = registerFeature("Enables support for advanced filters for VM 2.0 GA", "ROX_VULN_MGMT_ADVANCED_FILTERS", true)
```
Comment (dvail) [inline pkg/features/list.go:117]: Should we default these to `false` to avoid issues with nightlies/staging while these features are in development?

**Review by dvail — APPROVED** [review]
LGTM

My gut feeling is that we should have these flags start out as `false` since everything is a WIP, but let me know what you think.

File: tests/e2e/lib.sh [hunk 2]
```diff
@@ -143,7 +143,8 @@ export_test_environment() {
     ci_export ROX_VULN_MGMT_WORKLOAD_CVES "${ROX_VULN_MGMT_WORKLOAD_CVES:-true}"
     ci_export ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL "${ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL:-true}"
     ci_export ROX_VULN_MGMT_NODE_PLATFORM_CVES "${ROX_VULN_MGMT_NODE_PLATFORM_CVES:-true}"
-    ci_export ROX_VULN_MGMT_2_MISC_IMPROVEMENTS "${ROX_VULN_MGMT_2_MISC_IMPROVEMENTS:-true}"
+    ci_export ROX_VULN_MGMT_2_GA "${ROX_VULN_MGMT_2_GA:-true}"
+    ci_export ROX_VULN_MGMT_ADVANCED_FILTERS "${ROX_VULN_MGMT_ADVANCED_FILTERS:-true}"
```
Comment (pedrottimark) [inline tests/e2e/lib.sh:147]: There is another place recently added in this file. See step 4
https://github.com/stackrox/stackrox/tree/master/ui/apps/platform#add-a-feature-flag-to-frontend-code

File: pkg/features/list.go (hunk 1 above)
Comment (sachaudh) [reply pkg/features/list.go:117]: Sure we can do that

File: tests/e2e/lib.sh (hunk 2 above)
Comment (sachaudh) [reply tests/e2e/lib.sh:147]: Oh interesting, didn't know about this one. Could you explain this a bit more?

File: tools/allowed-large-files [hunk 3]
```diff
@@ -49,6 +49,7 @@ scripts/ci/lib.sh
 sensor/tests/data/policies.json
 sensor/upgrader/preflight/testdata/k8s-1.22.json.gz
 tests/performance/load/package-lock.json
+tests/e2e/lib.sh
```
Comment (sachaudh) [inline tools/allowed-large-files:52]: Looks like the file got too large 👀

File: tests/e2e/lib.sh (hunk 2 above)
Comment (pedrottimark) [reply tests/e2e/lib.sh:147]: For tests that install via operator instead of helm chart.

**Pull request merged**